x-pack/plugins/stack_alerts/kibana.json

```diff
@@ -14,7 +14,8 @@
     "triggersActionsUi",
     "kibanaReact",
     "savedObjects",
-    "data"
+    "data",
+    "kibanaUtils"
```

Comment:
@Dmitriynj seems `kibanaUtils` in not necessary? is it a leftover?

Reply:
I need it here 
```
import { Storage } from '@kbn/kibana-utils-plugin/public';
```

Reply:
oh, thx, missed that!